apps/definition_accept/lib/accept.ex

```diff
@@ -47,6 +46,12 @@ defmodule Accept do
             subset_id: nil,
             destination: nil,
             connection: nil
+
+  def on_new(ac) do
```

Comment:
Seems like a nice hook for something like this. Once we've got it working for all the events, I think we should talk about possibly changing it so the macro `new` does this. That would be dryer, but I think this feels cleaner / less surprising, so I may still lean this way even after we've looked at the alternative.

Reply:
I believe it would be relatively easy to add this functionality to Definition so all structs/events would get it for free, it could even support structs that don't have an id field.  This can be a refactor later, but I would consider it instead of updating all the events with this custom logic.  Let me know if you want details about how we can do this.

Reply:
Yeah, if it's not too tricky to do and also would support structs that don't have an id field, I think this is exactly what we should do. I wanted to go the current route first so at the least we'd have _something_ in the case that we do another team change / priority pivot etc. But I definitely think we should go the route you mention if we can.